app/javascript/react/utils/createGraphData.ts

```diff
@@ -0,0 +1,19 @@
+import { LatLngLiteral } from "../types/googleMaps";
+
+export const createFixedSeriesData = (data: any[]) =>
+  (data || [])
+    .map(({ time, value }: { time: number; value: number }) => [time, value])
+    .sort((a, b) => a[0] - b[0]);
+
+export const createMobileSeriesData = (data: any[], isGraphData: boolean) =>
```

Comment:
same as above, you can add a type instead of using any. Looking at the code, that would be 
`{
  time: number;
  value: number;
  lastMeasurementValue: number;
  point: { lat: number; lng: number };
  latitude: number;
  longitude: number;
};`

Reply:
and it looks like you might need to map the mobile measurements type to measurements type from measurementSlice